content/khalti-epayment.md

```diff
@@ -224,13 +225,25 @@ Sample of success response return URL.
 
 ### Sample Callback Request
 ```
-https://example.com/?pidx=EwGKrbdaYLTQ4rmWtNAMEJ
+https://example.com/payment?pidx=EwGKrbdaYLTQ4rmWtNAMEJ
 	&amount=1300
 	&mobile=98XXXXX403
 	&purchase_order_id=test12
 	&purchase_order_name=test
 	&transaction_id=MJbBJDKYziWqgvkgjxhS2W
 ```
+## Payment Failure Callback
+If, in-case, due to some problem, user transaction does not go through, the failure response is obtained in the return URL specified during payment initiate.
+Sample of failure response return URL. 
+
+- The callback url `return_url` should support `GET` method
+- User shall be redirected to the `return_url` with following parameters for confirmation
+    + message - _Failure message_
+
+### Sample Callback Request
+```
+https://example.com/payment?message=Could%20not%20process%20the%20payment.
```

Comment:
We need to work on this message. There should be atleast pidx in the argument list even in error

Reply:
Yes, we can add it in next build